jupyter-book/conditions/compositional-env.yaml

```diff
@@ -0,0 +1,20 @@
+name: compositional-env
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - r-base==4.0.5
+  - bioconductor-edger==3.32.1
+  - rpy2==3.4.2
+  - notebook
+  - schist==0.7.15
+  - pip
+  - pip:
+      - scanpy==1.9.1
+      - altair==4.2.0
+      - sccoda==0.1.8
+      - tasccoda==0.1.2
+      - toytree==2.0.1
+      - igraph==0.9.11 # required by schist.inference.nested_model
+      - scvi-tools==0.17.1
+      - git+https://github.com/emdann/milopy==0.1.0
```

Comment:
Didn't work for me

`git+https://github.com/emdann/milopy.git@master`

worked. `git+https://github.com/emdann/milopy.git@0.1.0` should as well

Reply:
yeah, sorry that's my bad. `master` should install the right version.